app/pb_kits/playbook/pb_selectable_icon/_selectable_icon.jsx

```diff
@@ -0,0 +1,107 @@
+/* @flow */
+
+import React from 'react'
+import classnames from 'classnames'
+import { Icon, Title } from '../'
+
+import {
+  buildAriaProps,
+  buildCss,
+  buildDataProps,
+} from '../utilities/props'
+
+type SelectableIconProps = {
+  aria?: Object,
+  checked?: Boolean,
+  className?: String,
+  disabled?: Boolean,
+  dark?: Boolean,
+  data?: Object,
+  icon: String,
+  inputId: String,
+  inputless?: Boolean,
+  multi?: Boolean,
+  name: String,
+  text: String,
+  value?: String
+}
+
+const SelectableIcon = ({
+  aria = {},
+  className,
+  checked = false,
+  dark = false,
+  data = {},
+  disabled = false,
+  icon,
+  inputId,
+  inputless = false,
+  multi = true,
+  name,
+  text,
+  value,
+  ...props
+}: SelectableIconProps) => {
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+
+  const css = buildCss({
+    'pb_selectable_icon_kit': true,
+    'checked': checked,
+    'dark': dark,
+    'disabled': disabled,
+    'enabled': !disabled,
+  })
+
+  const inputType = multi === false ? 'radio' : 'checkbox'
+
+  const inputIdPresent = inputId !== null ? inputId : name
+
+  return (
+    <div
+        {...ariaProps}
+        {...dataProps}
+        className={classnames(css, className)}
+    >
+      { inputless ? (
```

Comment:
I think the `<If condition={inputless}>` from [jsx-control-statements](https://github.com/AlexGilleran/jsx-control-statements#if) may help with improving code readability here.

Reply:
Ah, that's great.  Had seen this used elsewhere, didn't know we had access to this in our codebase.  Much cleaner than a long stringy ternary.

Reply:
Am seeing now it's just a babel feature... who knew?  Thanks for pointing this out!